content/en/tracing/setup_overview/setup/php.md

```diff
@@ -439,6 +439,213 @@ To remove the PHP tracer:
+...
+```
+
+Instructions follow in the next sections.
+
+#### Centos
+
+Install package `yum-utils` that provides the program `debuginfo-install`:
+
+```
+yum install -y yum-utils
+```
+
+Find the right package name for your PHP binaries, it can vary depending on the PHP installation method:
+
+```
+yum list installed | grep php
+```
+
+Install debug symbols. For example for package `php-fpm`:
+
+```
+debuginfo-install -y php-fpm
+```
+
+**Note**: if the repository that provides the PHP binaries is not enabled by default, it can be enabled when running the `debuginfo-install` command. For example:
+
+```
+debuginfo-install --enablerepo=remi-php74 -y php-fpm
+```
+
+#### Debian
+
+##### PHP installed from the Sury Debian DPA
+
+If PHP was installed from the [Sury Debian DPA][13], debug symbols are already available from the DPA. For example, for PHP-FPM 7.2:
+
+```
+apt update
+apt install -y php7.2-fpm-dbgsym
+```
+
+##### PHP installed from a different package
+
+The Debian project maintains a wiki page with [instructions to install debug symbols][14].
+
+Edit the file `/etc/apt/sources.list`:
+
+```
+# ... leave here all the pre-existing packages
+
+# add a `deb` deb http://deb.debian.org/debian-debug/ $RELEASE-debug main
+# For example for buster
+deb http://deb.debian.org/debian-debug/ buster-debug main
+```
+
+Update `apt`:
+
+```
+apt update
+```
+
+Try canonical package names for debug symbols, first. For example, if the package name is `php7.2-fpm` try:
+
+```
+apt install -y php7.2-fpm-dbgsym
+
+# if the above does not work
+
+apt install -y php7.2-fpm-dbg
+```
+
+In case debug symbols cannot be found yet, it is possible to use the utility tool `find-dbgsym-packages`. Install the binary:
+
+```
+apt install -y debian-goodies
+```
+
+Attempt finding debug symbols from either the full path to the binary or the process id of a running process:
+
+```
+find-dbgsym-packages /usr/sbin/php-fpm7.2
+```
+
+Install the resulting package name, if any has been found:
+
+```
+apt install -y php7.2-fpm-{package-name-returned-by-find-dbgsym-packages}
+```
+
+#### Ubuntu
+
+##### PHP installed from `ppa:ondrej/php`
+
+If PHP was installed from the [`ppa:ondrej/php`][15] edit the apt source file `/etc/apt/sources.list.d/ondrej-*.list` adding the `main/debug` component to it.
+
+Before:
+
+```deb http://ppa.launchpad.net/ondrej/php/ubuntu <version> main```
+
+After:
+
+```deb http://ppa.launchpad.net/ondrej/php/ubuntu <version> main main/debug```
+
+Update and install the debug symbols. For example, for PHP-FPM 7.2:
+
+```
+apt update
+apt install -y php7.2-fpm-dbgsym
+```
+##### PHP installed from a different package
+
+Find the right package name for your PHP binaries, it can vary depending on the PHP installation method:
+
+```
+apt list --installed | grep php
+```
+
+Note that in some cases `php-fpm` can be a metapackage that refers to the real package, for example `php7.2-fpm` in case of PHP-FPM 7.2. In this case the package name is the latter.
```

Comment:
```suggestion
**Note**: In some cases `php-fpm` can be a metapackage that refers to the real package, for example `php7.2-fpm` in case of PHP-FPM 7.2. In this case the package name is the latter.
```